index.html

```diff
@@ -0,0 +1,747 @@
+                
+            <div>
+                    <ul>
+                <li>
+                    <button class="Subscrib-now" type="button">SUBSCRIBE NOW
+                    </button>
+                </li>
+                <li>                    
+                        <button class="Log In" type="button">LOG IN</button>
+                    
+                </li>
+            
+            </ul>
+            </div>
+        </div>
+        </nav>
+    </header>
+     <main>
+        <div class="header">
+            <h1 class="heading">
+                Space Ripples Reveal Big Bang’s Smoking Gun
+            </h1>
+        </div>
+            <div >
+                <figure class="banner">
+                    <img src="assets/images/Alanglut.jpg" alt="Alan Guth was one of the first 
+                    physicists to hypothesize the 
+                    existence of inflation, which explains how the 
+                    universe expanded so uniformly and so quickly in 
+                    the instant after the Big Bang 13.8 billion years ago." width="960" >
+                    <figcaption>
+                        <span>
+                            Alan Guth was one of the first physicists
+                            to hypothesize the existence of inflation, which explains how the
+                            universe expanded so uniformly and so
+                            quickly in the instant after the Big Bang 13.8 billion years ago.
+                        </span>
+                        <span>Rick Friedman for The New York Times</span>
+                    </figcaption>
+
+                </figure>
+        </div>
+        <section>
+            <div class=author-details>
+                <h4 class=author>By Dennis Overbye</h4>
+                <div class="flex-display">
+                <p><time datetime="2014-03-17 20:00">March 17, 2014</time>
+                    <p style="margin-right: 22rem;">
+                    <ul style="display: flex;">
+                        <li><a href="#"><svg class="svg" viewBox="0 0 7 15" width="7" height="15">
+                            <path fill-rule="evenodd" clip-rule="evenodd"
+                                d="M4.775 14.163V7.08h1.923l.255-2.441H4.775l.004-1.222c0-.636.06-.977.958-.977H6.94V0H5.016c-2.31 0-3.123 1.184-3.123 3.175V4.64H.453v2.44h1.44v7.083h2.882z"
+                                fill="#000"></path>
+                        </svg></a></li>
+                        <li><a href="#"><svg class="svg" viewBox="0 0 13 10" width="13" height="10">
```

Comment:
Please, remove inline styling, it is considered a bad practice, move it to your CSS files.